build_config.rb

```diff
@@ -82,7 +82,7 @@
   conf.gem 'deps/mruby-file-stat'
   conf.gem 'deps/mruby-io'
   conf.gem 'deps/mruby-nanovg'
-  conf.gem 'deps/mruby-process'
+#  conf.gem 'deps/mruby-process'
```

Comment:
ok, so tracking the dependencies around it looks like mruby-process is used by mruby-dir-glob (for IIRC some rather dumb reasons). So, in order to keep the build one which does not try fetching things over the network we need to have this dep still in place, though there is a chance it might be something that can be ignored on windows. I'd have to double check though.

Reply:
I just find that there's no need to use this dependency at all.
Besides, bypassing `mruby-process` within `mruby-dir-glob` is also a good choice, see below (`zyn-fusion-build/mruby-dir-glob-no-process.patch`):

```diff
diff --git a/mrbgem.rake b/mrbgem.rake
index e7481ab..b6c9d70 100644
--- a/mrbgem.rake
+++ b/mrbgem.rake
@@ -8,9 +8,9 @@ MRuby::Gem::Specification.new('mruby-dir-glob') do |spec|
   spec.add_dependency 'mruby-dir'
   spec.add_dependency 'mruby-io'
   spec.add_dependency 'mruby-errno'
-  spec.add_dependency 'mruby-file-stat'
-  spec.add_dependency 'mruby-process'
-  spec.add_dependency 'mruby-pack'
+  spec.add_test_dependency 'mruby-file-stat'
+  # spec.add_test_dependency 'mruby-process'
+  spec.add_test_dependency 'mruby-pack'
 
   # how to specify a 'general' regexp engine?
 #  spec.add_dependency 'mruby-regexp-pcre'

```

It works well on all those targets: Msys2, Linux native build as well as Linux cross-build.